recipes/abismal/meta.yaml

```diff
@@ -24,8 +24,10 @@ requirements:
     - llvm-openmp  # [osx]
     - libgomp      # [linux]
   host:
+    - zlib
     - htslib
   run:
+    - zlib
```

Comment:
According to https://bioconda.github.io/faqs.html?highlight=zlib#how-does-global-pinning-work this is not really needed!
```suggestion
```